### DISCUSSION TITLE
Addition To REFramework Possibility

### DISCUSSION THREAD

**Discussion by Adore33** [praydog/REFramework#369] [General Questions]
First off, thank you Praydog for your continued updates.

I have been using hex edits of the .exe for both RE2:R and RE3:R in order to fix the zombies animating at 30FPS in the distance instead of 60FPS.
Been getting them from "residentevilmodding.boards.net" as they are available.
Is a Zombie FPS Fix something that could be implemented into the REFramework for these two games?

Thank You.

**Comment by praydog** [comment]
Yes. I can look into it.

**Reply by Adore33** [reply]
> Yes. I can look into it.

Thank you for taking the time to respond and for considering the addition.
